pkgs/development/python-modules/pypdfium2/default.nix

```diff
@@ -37,6 +40,13 @@ let
       hash = "sha256-3JA7cW/xaEj/DxMHEypROwrKGo7EwUEcipRqALTvydw=";
     };
 
+    # override auto-detection of cc, fixes cross
+    postPatch = ''
+      substituteInPlace src/ctypesgen/__main__.py \
+        --replace-fail 'if args.cpp:' 'if True:' \
```

Comment:
I'd rather have it as a patch, for readability primarily, but also because this replace rule may apply wider than we may hope.